game/scripts/npc/items/item_allied_cyclone.txt

```diff
@@ -100,7 +99,63 @@
         "cast_range_tooltip"                              "575"
       }
     }
-  }
+      "OnSpellStart"
+    {
+
+      "ApplyModifier"
+      {
+        "Target"                                          "TARGET"
+        "ModifierName"                                    "modifier_item_allied_cyclone_active"
```

Comment:
I think you might be able to just apply the in-built modifier. It's called `modifier_eul_cyclone`.